RangeOutputstream is missing 1 byte

Der-Jan opened the pull request:
When playing around with HTML5 Audio tag as a replacement for JWPlayer, my iPad never started playing. Usign Wireshark I saw, it is sending a Range: bytes=0-1 HTTP header. Subsonic only sent back byte 0 (instead of byte 0 and byte 1), so the iPad was waiting for the second byte.

Commit:
Use id in web interface.

Commit:
Use id in web interface.

Commit:
Use id in video player.

Commit:
Remove "path" from playAddDownload.jsp

Commit:
Remove usage of "pathUtf8Hex" in code.

Commit:
Web support for starring individual songs in playlist.

Commit:
Use ratingOn/Off rather than star image.

Commit:
Web page for showing starred stuff.

Commit:
Fixed documentation of REST createUser.

Commit:
Added REST methods getStarred and getStarred2

Commit:
Use id rather than path in tag editing.

Commit:
Added playlist tables.

Commit:
Renamed Playlist to PlayQueue.

Commit:
Added Playlist and PlaylistDao.

Commit:
Reimplemented REST playlist methods.

Commit:
Reimplemented playlist feature in PodcastController.

Commit:
Load playlist.

Commit:
Save playlist.

Commit:
Save playlist.

Commit:
Save playlist.

Commit:
Updated todo.

Commit:
Remove deprecated methods in PlaylistService.

Commit:
Rename playlist.jsp to playQueue.jsp

Commit:
Rename PlaylistController.

Commit:
Add last_updated to playlist table.

Commit:
Show playlists in left.jsp

Commit:
Rename playlist_song to playlist_file

Commit:
Rename playlist.last_modified to playlist.changed

Commit:
Rename media_file.last_modified to media_file.changed

Commit:
Fixed a bug in left.jsp

Commit:
Made PlaylistController

Commit:
playlist.jsp

Commit:
playlist.jsp

Commit:
Rename PlaylistInfo to PlayQueueInfo.

Commit:
playlist.jsp

Commit:
playlist.jsp

Commit:
playlist.jsp

Commit:
playlist.jsp

Commit:
playlist.jsp

Commit:
playlist.jsp

Commit:
playlist.jsp

Commit:
Load playlist in left.jsp dynamically.

Der-Jan committed:
Fixed Range handling

Der-Jan committed:
Rudimentary HTML5 Audio Tag

Commit:
Update playlists in left.jsp

Commit:
Save queue as playlist.

Commit:
Added jquery ui

Commit:
Use jquery in playlist.jsp

Commit:
Confirmation dialog when deleting a playlist.

Commit:
Option to change name, comment, public.

Commit:
playQueue refactoring

Commit:
Rewrite playQueue.jsp to use jquery.

Commit:
Append to playlist.

Commit:
Append to playlist.

Commit:
Append to playlist.

Commit:
Use jquery in home.jsp

Commit:
Use jquery in main.jsp

Commit:
Use jquery in starred.jsp

Commit:
Remove scriptaculous.

Commit:
Remove loadPlaylist.jsp etc

Commit:
Make append to playlist work in main.jsp

Commit:
Make it easy to create a new empty playlist.

Commit:
Only show editable playlists when appending.

Commit:
REST support, add/deletePlaylistUser.

timoreimann commented:
Do I understand commit 7171f4d right that it completely removes JWPlayer? If so, what's the reason for that? Shouldn't your Range handling modification (commit 63da509) alone fix the bug you are describing?

Commit:
Updated Estonian translation, courtesy of Olav Mägi.

Commit:
Updated Catalan translation, courtesy of Josep Santalo.

Commit:
Remove settings for playlist folder.

Commit:
Remove playlist role.

Der-Jan commented:
The bug should be fixed with commit 63da509 - removing JWPlayer is not directly related - I want to be able to use my iPhone to play the music without an app. JWPlayer (in HTML5 mode) doesn't seem to be a good option for this, since it's using `<video>` tag instead of the `<audio>`.

Commit:
REST: Return shared users for playlist.

timoreimann commented:
I feel somewhat reluctant to accept the pull request yet if one of the commits isn't needed to fix the bug _and_ removes JWPlayer from the code base. Admittedly, my knowledge about HTML5 audio/video playing is limited, so I need to ask you this directly: What's the implication of removing JWPlayer? Will it break compatibility with certain browsers?

Additionally, issue #32 is dealing with introducing HTML5 playback as well. @porkcharsui: You're more involved with the subject than I am. What's your opinion on this?

porkcharsui commented:
I'll review this tomorrow since I need to evaluate the change's impacts with media playback. The RangeOutputStream fix is a good find! 

Eventually I want to replace the UI as a whole (player, UI key bindings, etc) while allowing fall back to the "Subsonic" frame interface if needed.

Commit:
Updated todo

Commit:
Escape some chars in podcast.
http://forum.subsonic.org/forum/viewtopic.php?f=2&t=5717&p=21988&hilit=podcast#p21988

Commit:
REST: Create updatePlaylist (for changing metadata, removing/adding songs/users).

Commit:
REST: Optional username parameter in getPlaylists method

Commit:
Nicer star icon.

Commit:
Improve documentation of REST "download" method.

Der-Jan committed:
Disable Range requests while transcoding - otherwise oga for Firefox gets screwed up

Der-Jan committed:
First transcode to get complete length

Commit:
Update playlist xml examples.

Der-Jan committed:
No transcoding, if already in preferred format

Der-Jan committed:
Transcoding is now defined by browser (by adding supported mimetype)

Der-Jan committed:
Added webm as audio format

Der-Jan committed:
Transcode first, to have the transcoded file size

Commit:
Save album path.

Commit:
Improve album resolution

Der-Jan committed:
Removed double webm

Commit:
Album resolution

Commit:
Album resolution

Der-Jan committed:
?

Commit:
Change "Play queue is empty" to "Playlist is empty"

Commit:
Create two album list types: alphabeticalByTitle, alphabeticalByArtist.

Commit:
Create two album list types: alphabeticalByTitle, alphabeticalByArtist.

Commit:
Update TODO

Commit:
Document new REST methods in api.jsp

Commit:
Refactored api.jsp

Commit:
Documented getAlbumList2.

Commit:
Documented search3.

Commit:
Documented updatePlaylist.

Commit:
Documented star methods.

Commit:
Fixed typo

Commit:
Document new REST methods in api.jsp

Commit:
Document new REST methods in api.jsp

Commit:
Document new REST methods in api.jsp

Der-Jan committed:
Merge branch 'master' of https://github.com/Mach5/supersonic

Commit:
Document new REST methods in api.jsp

Commit:
Document new REST methods in api.jsp

Der-Jan committed:
Added status to getTranscodedInputStream, currently not working. WeakHashMap doesn't work neither.

Der-Jan committed:
Committed .swp by accident

Der-Jan committed:
Updated to latest commons-io

Der-Jan committed:
Trying to make transcoding work - Parameter.equals still doesn't work

Der-Jan committed:
Reformated MediaFile equals for debugging

Der-Jan committed:
Working cache - something's still odd

Der-Jan committed:
Added transcoding for ogg and webm

Der-Jan committed:
Only check formats active for player

Der-Jan committed:
Switch transcoding bitrate to 320kbps

Der-Jan committed:
Avoid stopping of playback, when adding or removing songs - however the play button still doesn't really work (e.g. after reloading the page)

Commit:
Updated exe4j

Commit:
Updated XO and Chumby apps.

Commit:
Updated XO and Chumby apps.

Commit:
Updated todo

Commit:
Limit functionality of REST playlist mgmt.

Commit:
Use port 4040/4443 on windows.
Increase memory limit to 150MB.

Commit:
Update changelog.jsp

Commit:
Fixed download.jsp

Commit:
Updated todo

Commit:
Updated todo

timoreimann commented:
@porkcharsui, I took the liberty to assign this issue to you for the moment because you seem most qualified to review the impact of the pull request.

Commit:
Fixed bug in truncate()

Commit:
Support adding directory root (c:\) as music folder.

Commit:
Guess year when creating albums.

Commit:
Updated todo.

Commit:
Added Czech translation, courtesy of Robert Ilyk.

Commit:
Show album year.

Commit:
Sort albums by year.

Commit:
Import playlist.

Commit:
Playlist import.

Commit:
Reduce flashing refresh of home page.

Commit:
Display directory name rather than album name?
    http://forum.subsonic.org/forum/viewtopic.php?f=4&t=9411&p=41514#p41514
"R.E.M" displayed as "R.E"
  http://forum.subsonic.org/forum/viewtopic.php?f=4&t=9411&p=41790#p41790

Commit:
Playlist export.

Commit:
Genre search problem.
  http://forum.subsonic.org/forum/viewtopic.php?f=4&t=9411&p=41839#p41804

Commit:
Made CoverArtController more robust.

Commit:
Show message if nothing is starred.

Commit:
Better error message if media file or playlist not found.

Commit:
Added Hypersonic app.

Commit:
Added Hypersonic app.

Commit:
Updated Periscope app.

Commit:
Updated to 4.7.beta2

Commit:
Updated changelog

Commit:
Fixed typo on apps page.

Commit:
Don't display "null" as artist in now playing.

Commit:
Made nsis executable

Commit:
Fixed crash on Java 7 caused by wrong sorting implementation.

Commit:
Added Settings > General > Sort albums by year

Commit:
Try with different encodings when importing playlists.

Commit:
Fixed UT.

Commit:
Admins should be able to manage all playlists (that is visible to them)

Commit:
Use "album artist" tag if present.

Commit:
Updated Slovenian translation, courtesy of Jan Jamsek

Commit:
Automatically import all playlists on start-up.
Show all playlist metadata.

Commit:
Fixed problem with no files being showed if removing and re-adding a media folder with scans in between.

Commit:
Renamed PlaylistInputStream.

Commit:
Renamed RandomPlaylistController.

Commit:
Added "Settings > Media folders > Clean-up database"

Commit:
Updated TODO.

Commit:
Updated todo

Commit:
Fixed slow "Top rated".

Commit:
Updated Google Analytics tracking code

Commit:
Less aggressive caching of left.view

Commit:
Moved library statistics to the bottom.

Commit:
Collapse long list of playlists.

Commit:
Make it possible to refresh left frame.

Commit:
Refactored overloaded getChildrenOf() methods.

Commit:
Displayed album year doesn't update when editing corresponding tag (Foyaxe)

Commit:
Update documention wrt port 4040 on Windows.

Commit:
Added MiniSub.

Commit:
Added MiniSub

Commit:
Updated MiniSub.

Commit:
Added minisub to More page.

Commit:
Show toast when saving/updating playlist from play queue.

Commit:
Show toast when saving settings.

Commit:
Show toast when saving settings.

Commit:
Show toast when saving settings.

Commit:
Show toast when saving settings.

Commit:
Use jquery in search.jsp

Commit:
REST: Added getUsers.

Commit:
REST doc: create getVideos example.

sindremehus committed:
Upgraded to 4.7.beta3

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3051 104adb81-38d2-4886-885e-82898e39ce25

Commit:
Upgraded to 4.7.beta3

sindremehus committed:
HLS support.


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3052 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Moved "Downsample command" to Settings > Transcoding.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3053 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Added "HLS command" to Settings > Transcoding.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3054 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Set proper content type for mpeg ts.


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3055 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Use relative URIs in the HLS playlist.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3056 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
HLS:   Support multiple bitrates.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3057 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
HLS: REST doc.


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3058 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
HLS: Support adaptive bitrate streaming.


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3059 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Add default hls and downsample command to web pages.


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3060 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Use explicit columns when populating transcoding2.


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3061 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Album art broken on IE8.


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3062 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Fixed HLS urls after running mediastreamvalidator.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3063 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Rename Subsonos to MMTM.


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3064 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Refactored apps page.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3065 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Add minisub chrome app to apps.jsp


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3066 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Refactor MusicIndexService

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3067 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Fixed performance of media scanning (missing index in album table)

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3068 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
getStatistics is slow.


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3069 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
REST: getArtists now only returns album artists.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3070 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Use mp3 in hls.


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3071 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
REST: Add index to getArtists


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3072 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Improve markNonPresent() etc

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3073 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Added new theme "SubStandard", courtesy of Dan Eriksen.


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3074 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
HLS audio support.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3075 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Added RESTFilter for intercepting exception in RESTController.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3076 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Simplified exception handling in RESTController.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3077 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Update changelog for beta3


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3078 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Backup database in installer.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3079 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Updated todo

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3080 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Backup database in installers (linux, mac).


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3081 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Fixed bug in subsonic.nsi.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3082 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Tagging 4.7.beta3

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/tags/release-4.7.beta3@3083 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Updated minisub text

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3084 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Updated todo

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3085 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Updated subStandard.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3086 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Updated svn:ignore

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3087 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Avoid NPE in ShareService.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3088 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Updated todo.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3089 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Fix scrolling bug in playlist.jsp and playQueue.jsp

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3090 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Fixed wrong minisub link.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3091 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Changed wording of public playlists.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3092 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Fixed bug in genre searching.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3093 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Updated POM with new svn url.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3094 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Bugfix: Rescan looses comments and play statistics.


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3095 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Updated todo.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3096 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Auto-import playlists after scan.


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3097 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Refactor PlaylistService.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3098 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Refactor PlaylistService.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3099 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Started work on normalizing paths in playlist import.

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3100 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
PlaylistService.normalizePath()

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3101 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Update playlist statistics after rescan.


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3102 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Upgraded to JW Player 5.10


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3103 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
REST: Add timestamp to scrobble method, and support multiple entries.


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3104 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Upgraded to 4.7


git-svn-id: svn://svn.code.sf.net/p/subsonic/code/trunk@3105 104adb81-38d2-4886-885e-82898e39ce25

sindremehus committed:
Tagging 4.7

git-svn-id: svn://svn.code.sf.net/p/subsonic/code/tags/release-4.7@3106 104adb81-38d2-4886-885e-82898e39ce25

Der-Jan committed:
Changed from folder to cover in security check

Der-Jan committed:
Corrected URL

Der-Jan committed:
Merged with Mach5

Der-Jan committed:
Merge branch 'subsonic.svn-4.7.beta3' into subsonic.upstream

Conflicts:
	subsonic-installer-debian/src/DEBIAN/preinst
	subsonic-installer-mac/src/preinstall.sh
	subsonic-installer-rpm/src/subsonic.spec
	subsonic-installer-windows/src/main/nsis/subsonic.nsi
	subsonic-main/Getting Started.html
	subsonic-main/TODO.TXT
	subsonic-main/src/main/java/net/sourceforge/subsonic/command/AdvancedSettingsCommand.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/controller/AdvancedSettingsController.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/controller/LeftController.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/controller/RESTController.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/controller/StreamController.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/controller/TranscodingSettingsController.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/controller/WapController.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/dao/AlbumDao.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/dao/ArtistDao.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/dao/MediaFileDao.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/dao/schema/Schema.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/dao/schema/Schema46.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/dao/schema/Schema47.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/domain/MediaLibraryStatistics.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/domain/MusicIndex.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/domain/VideoTranscodingSettings.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/service/JukeboxService.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/service/MediaScannerService.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/service/MusicIndexService.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/service/SettingsService.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/service/TranscodingService.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/util/StringUtil.java
	subsonic-main/src/main/resources/net/sourceforge/subsonic/i18n/ResourceBundle_en.properties
	subsonic-main/src/main/resources/net/sourceforge/subsonic/theme/coolandclean.properties
	subsonic-main/src/main/resources/net/sourceforge/subsonic/theme/themes.txt
	subsonic-main/src/main/webapp/WEB-INF/applicationContext-security.xml
	subsonic-main/src/main/webapp/WEB-INF/applicationContext-service.xml
	subsonic-main/src/main/webapp/WEB-INF/jsp/advancedSettings.jsp
	subsonic-main/src/main/webapp/WEB-INF/jsp/coverArt.jsp
	subsonic-main/src/main/webapp/WEB-INF/jsp/test.jsp
	subsonic-main/src/main/webapp/WEB-INF/jsp/transcodingSettings.jsp
	subsonic-main/src/main/webapp/WEB-INF/subsonic-servlet.xml
	subsonic-main/src/main/webapp/WEB-INF/web.xml
	subsonic-main/src/main/webapp/xsd/artists_example_1.xml
	subsonic-main/src/main/webapp/xsd/subsonic-rest-api.xsd
	subsonic-site/src/main/webapp/api.jsp
	subsonic-site/src/main/webapp/apps.jsp
	subsonic-site/src/main/webapp/changelog.jsp
	subsonic-site/src/main/webapp/transcoding.jsp

Der-Jan committed:
Merge branch 'subsonic.svn-4.7' into subsonic.upstream

Der-Jan committed:
Typo when merging

Der-Jan committed:
Removed license check

Der-Jan committed:
Merge branch 'subsonic.upstream.4.7'

Conflicts:
	pom.xml
	subsonic-assembly/pom.xml
	subsonic-backend/pom.xml
	subsonic-backend/src/main/java/net/sourceforge/subsonic/backend/controller/MultiController.java
	subsonic-booter/pom.xml
	subsonic-booter/src/main/script/subsonic.bat
	subsonic-booter/src/main/script/subsonic.sh
	subsonic-installer-debian/pom.xml
	subsonic-installer-debian/src/etc/default/subsonic
	subsonic-installer-debian/src/etc/default/supersonic
	subsonic-installer-mac/pom.xml
	subsonic-installer-rpm/pom.xml
	subsonic-installer-rpm/src/subsonic.spec
	subsonic-installer-windows/pom.xml
	subsonic-installer-windows/src/main/exe4j/supersonic-service.exe.vmoptions
	subsonic-installer-windows/src/main/nsis/supersonic.nsi
	subsonic-main/Getting Started.html
	subsonic-main/pom.xml
	subsonic-main/src/main/java/net/sourceforge/subsonic/command/NetworkSettingsCommand.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/controller/PlaylistController.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/controller/StreamController.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/dao/schema/Schema46.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/io/PlayQueueInputStream.java
	subsonic-main/src/main/java/net/sourceforge/subsonic/service/SettingsService.java
	subsonic-main/src/main/webapp/WEB-INF/jsp/main.jsp
	subsonic-main/src/main/webapp/WEB-INF/jsp/networkSettings.jsp
	subsonic-main/src/main/webapp/WEB-INF/jsp/playlist.jsp
	subsonic-main/src/main/webapp/WEB-INF/jsp/right.jsp
	subsonic-main/src/main/webapp/WEB-INF/jsp/videoPlayer.jsp
	subsonic-site/pom.xml
	subsonic-site/src/main/webapp/api.jsp
	subsonic-site/src/main/webapp/apps.jsp
	subsonic-site/src/main/webapp/changelog.jsp
	subsonic-site/src/main/webapp/download.jsp
	subsonic-site/src/main/webapp/getting-started.jsp
	subsonic-site/src/main/webapp/installation.jsp
	subsonic-site/src/main/webapp/tracking.jsp
	subsonic-site/src/main/webapp/transcoding.jsp
	subsonic-web/version.html

Der-Jan committed:
Merge broke pom.xml
